### ISSUE TITLE
Recording: new pipeline 

### ISSUE THREAD

**Issue by tpoisot** [GEO-BON/boninabox-pipelines#117]
Based on a few discussions post demo yesterday, I think it would be useful to do a little series of videos to show how to implement a script/pipeline.

I'm volunteering to do this with the bioclim SDM as an example (just because it's trivial to write).

Feedback?

**Comment by jmlord** [comment]
Yes, would be useful indeed! I had planned to do a series of examples of incremental complexity, directly in the pipeline editor, but didn't have the time. 

Videos would be a good option too, we could host them on boninabox.geobon.org or create a dedicated youtube channel. What do you think?

**Comment by frousseu** [comment]
I thought about that too at the workshop. Guillaume said there was already a video (but 45 min?), but I think shorter and more targeted videos would be helpful.

**Comment by jmlord** [comment]
Jory has made these videos in another context. They are accessible on YouTube and linked from inside our documentation.
Channel: https://www.youtube.com/watch?v=mLjnjCWwU3w&list=PLND0FZa0b1hL4susdnnA9wsRKfrWdPt1v
How to write a script: https://youtu.be/nvYMC_xuOpE?si=dmn5cLuTHjXe1tiW
How to create a pipeline: https://youtu.be/LwncUuc4WeM?si=Yw0vmdJ80XId5S1n
